patches/lottie-react-native+6.5.1.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/node_modules/lottie-react-native/ios/Fabric/LottieAnimationViewComponentView.mm b/node_modules/lottie-react-native/ios/Fabric/LottieAnimationViewComponentView.mm
+index a4be628..d8f304e 100644
+--- a/node_modules/lottie-react-native/ios/Fabric/LottieAnimationViewComponentView.mm
++++ b/node_modules/lottie-react-native/ios/Fabric/LottieAnimationViewComponentView.mm
+@@ -40,6 +40,11 @@ - (instancetype) initWithFrame:(CGRect)frame
+ 
+ #pragma mark - RCTComponentViewProtocol
+ 
+++ (BOOL)shouldBeRecycled
```

Comment:
don't see an upstream PR for this change

Reply:
@WoLewicki @roryabraham this patch cause LottieView autoplay prop not working, it always autoplay, there is no reported issue since all of the lottie assets autoplay

Reply:
@wildan-m  could you be more precise why do you suspect this change makes `autoplay` prop not working? This should not affect props that are sent to the component but I may be missing something.

Reply:
@WoLewicki in IOS you could try to make autoplay={false}, and it keep auto playing, but after removing the patch it works as expected

Reply:
Could you make a quick repo with a small repro of this issue? This code should only make the props not to be transferred between the instances of the view, so if you set it correctly on each instance, it should be applied. From top of my head I can only think of an example where you create new instances of the view and don't set the `autoplay` prop explicitly, then it would use the previous value. Here: https://github.com/lottie-react-native/lottie-react-native/blob/c604757f797d977c8dd025e66ffa2f91e60869a6/packages/core/ios/Fabric/LottieAnimationViewComponentView.mm#L97 it should be set correctly based on the value it got.